ccpage166.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+# Make a class called Restaurant.
+# The __init__() method for Restaurant should store two attributes: a restaurant_name and a cuisine_type.
+# Make a method called describe_restaurant() that prints these two pieces of information,
+# and a method called open_restaurant() that prints a message indi- cating that the restaurant is open.
+# Make an instance called restaurant from your class . Print the two attri- butes individually, and then call both methods.
+
+class Restaurant():
+    """An attempt to model a restaurant"""
+    def __init__(self, name, cuisine):
+        self.name = name
+        self.cuisine = cuisine
+
+    def describe_restaurant(self):
+        """Describe the restaurant"""
+        print(f"{self.name.title()} is the name of the restaurant.")
+        print(f"{self.name.title()} serves {self.cuisine.title()} food.")
+
+    def open(self):
+        """Indicate whether the restaurant is open"""
+        print(f"{self.name.title()} is open today.")
+
+restaurant = Restaurant('grotto', 'italian')
+
+
+print(f"My favorite restaurant is called {restaurant.name.title()}.")
+print(f"{restaurant.name.title()} serves {restaurant.cuisine.title()} food.")
+
+restaurant.describe_restaurant()
+restaurant.open()
+
+
+# Start with your class from Exercise 9-1 (exercise listed above).
+# Create three different instances from the class, and call describe_restaurant() for each instance.
+class Restaurant():
```

Comment:
You _could_ have used your class from above instead of redoing it, but that's fine.

Reply:
I'd personally remove the second definition before you merge this.